### ISSUE TITLE
UML Attributes are not visible on Linux CentOS 9

### ISSUE THREAD

**Issue by mkwapisz** [ModelioOpenSource/Modelio#54]
CentOS Stream 9
Gnome 40.4.0
Weyland windows manager

As presented on the following picture UML attributes are not visible for classes (operations and attributes as well)

![obraz](https://user-images.githubusercontent.com/24523507/225575846-e484bc56-0f0a-4126-9bbc-18fb9351ac58.png)

**Comment by mvdw** [comment]
IIUC, you mean: "the UML properties panel", in stead of "UML attributes"

**Comment by IrinaNeculita** [comment]
Hello,
If the operations/attributes are created in explorer you have to unmask them by drag and drop in diagram. 
The problem is reported to developers.

**Comment by mkwapisz** [comment]
@mvdw yes, the property panel is not visible. You cannot edit them. This is how it looks on Windows.
![2023-04-27 121951 Modelio](https://user-images.githubusercontent.com/24523507/234834611-cc4d22f1-c3d6-49cd-ab2c-f46693fc5a31.png)
